source/Lib/DecoderLib/DecCu.cpp

```diff
@@ -169,8 +169,8 @@ void DecCu::TaskDeriveDMVRMotionInfo( CodingStructure& cs, const UnitArea& ctuAr
       const int dy = std::min<int>( pu.lumaSize().height, DMVR_SUBCU_HEIGHT );
       const int dx = std::min<int>( pu.lumaSize().width,  DMVR_SUBCU_WIDTH );
       
-      static constexpr unsigned scale = 4 * std::max<int>(1, 4 * AMVP_DECIMATION_FACTOR / 4);
-      static constexpr unsigned mask  = scale - 1;
+      static const unsigned scale = 4 * std::max<int>(1, 4 * AMVP_DECIMATION_FACTOR / 4);
```

Comment:
I guess now the standard is finalized, this can be just written out as constexpr unsigned scale = 8. The way it was defined was just for standard development purposes in VTM.

Will push it with the next update.

Closing the pull request.